controller/volume_controller.go

```diff
@@ -1000,6 +1000,7 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, es map[stri
 					rs[r.Name] = r
 				}
 			}
+			v.Status.Robustness = types.VolumeRobustnessFaulted
```

Comment:
Do we need to set the volume state here?

Reply:
Yes, we need to set it so that Longhorn sets `v.Status.RemountRequestedAt` in the next reconciliation run at https://github.com/longhorn/longhorn-manager/blob/95b63e85741c189bdae2112cf2f2fcf7df8f8a50/controller/volume_controller.go#L981.

It also makes sense because when the engine crash, the volume should be faulty.